projects/angular/src/emphasis/alert/_alert.clarity.scss

```diff
@@ -175,42 +175,49 @@
     flex: 1 1 auto;
     display: flex;
     flex-wrap: nowrap;
-    min-height: alert-variables.$clr-alert-item-min-height;
-    margin-bottom: variables.$clr_baselineRem_0_25;
+    @include mixins.css-var(gap, clr-alert-items-gap, 0, variables.$clr-use-custom-properties);
 
+    min-height: alert-variables.$clr-alert-item-min-height;
+    @include mixins.css-var(
+      margin-bottom,
+      clr-alert-item-margin-bottom,
+      variables.$clr_baselineRem_0_25,
+      variables.$clr-use-custom-properties
+    );
     &:last-child {
       margin-bottom: 0;
     }
   }
 
   .alert-items {
-    $vertPadding: variables.$clr_baselineRem_10px;
-    $horizPadding: variables.$clr_baselineRem_0_5 - alert-variables.$clr-alert-borderwidth;
     flex: 1 1 auto;
     flex-flow: column nowrap;
-    padding: $vertPadding $horizPadding;
+    @include mixins.css-var(
+      padding,
+      clr-alert-padding,
+      variables.$clr_baselineRem_10px variables.$clr_baselineRem_0_5 - alert-variables.$clr-alert-borderwidth,
+      variables.$clr-use-custom-properties
+    );
     display: flex;
   }
 
-  //display: inline-block and max-width were specifically added for IE 10.
-  //Flexbox content wouldn't wrap otherwise :(. 98% was just an estimate to distance the text from the
-  //close alert button.
   .alert-item > span,
   .alert-text {
-    display: inline-block; //needed for IE11
-    flex-grow: 1;
+    @include mixins.css-var(flex-grow, clr-alert-text-flex-grow, 1, variables.$clr-use-custom-properties);
     flex-shrink: 1;
-    flex-basis: 98%; //needed for IE11
-    max-width: 98%; //needed for IE11
     margin-right: variables.$clr_baselineRem_0_5;
```

Comment:
Why do we keep this margin in token migration? 
The spacing is done by `gaps`. Keep it only for NG theme.


```suggestion
```